ils_middleware/tasks/amazon/bluecore_records_s3.py

```diff
@@ -0,0 +1,31 @@
+import logging
+import pathlib
+
+import httpx
+from airflow.providers.amazon.aws.hooks.s3 import S3Hook
+
+logger = logging.getLogger(__name__)
+
```

Comment:
Is there a reason why we prefer accepting any kwargs in these tasks rather the specifying what they should be?

Reply:
For the `get_file` function, there are different kwargs depending on the type of file. Normally for Airflow tasks I typically use kwargs instead of fixed args to allow for retrieving the Airflow specific variables like `ti` (the task instance) in the future. Happy to change these to parameters instead.